bentoml/server/gunicorn_server.py

```diff
@@ -96,7 +97,7 @@ def __init__(
 
             self.options = {
                 "bind": self.bind,
-                "timeout": timeout,
+                "timeout": timeout,  # TODO
```

Comment:
what's the reason for adding a TODO here? I thought it was passed here via the config already?

Reply:
I'm thinking about verifying the values between timeout and max_latency.